submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,105 @@
+
+[Dedicated mall example from community]
+
+## **Which recipes should be included?**
+
+The ideal mall would include a machine for practically everything, but it would be huge and resource-intensive. It makes more sense to build according to what you really need. For a starter mall, I would recommend including the first-tier items you will need all the time when building the factory: assembling machines, inserters, belts, underground belts, and splitters. The beautiful thing about all of these items is that they are ingredients for their next tiers. As a result, overproducing them is rarely wasteful and you can easily handcraft second-tier items when you have a decent supply of them and intermediate items. For example, with splitters and intermediates in hand, a fast splitter takes only two seconds to craft instead of nearly 30 seconds when using raw materials. Other items that are great to pick up from an early-game mall are gears, electronic circuits, long-handed inserters, pipes, firearm magazines, gun turrets, and power poles. If you want to plan in advance for the mid game, it would be useful for your starter mall to have empty machines reserved for making engines (for vehicles and pumps) and the most common second-tier items: assembling machine 2's, fast inserters, and fast belts..
+
+Later into the game, you develop a demand for groups of items for specific purposes like oil processing, railway systems, modules, armor equipment, and armaments. These item groups can perhaps be added to an expanded starter mall or maybe have malls of their own. 
+
+[Visual of item groups: starter items, production items, oil processing items, rail system items, etc.]
+
+## How many machines should each recipe have?
+
+Mall assembling machines are *usually* idle more often than not because usually one does not build without interruption. Therefore ratios don't matter nearly as much as they usually do and often one machine per recipe is enough. 
+
+The exception to this rule is highly-used intermediates that would become serious bottlenecks. Namely, you would benefit from having a few assembling machines for electronic circuits and several machines for iron gear wheels (especially when making higher-tier transport belts). Meanwhile, for high-tech equipment and for modules, you need large quantities of expensive intermediate products such as advanced circuits, processing units, and low density structures. Therefore a mall for such items becomes difficult to supply using raw materials alone and it is worth building separate dedicated areas to craft and supply these expensive intermediates. 
+
+[visual idea?]
```

Comment:
a general construction mall and on another place, a dedicated module production plant?